data.sql

```diff
@@ -11,3 +11,27 @@ VALUES ('Charmander','2020-02-08', 0, '1', -11),
        ('Boarmon','2005-06-07', 7, '1', 20.4),
        ('Blossom','1998-10-13', 3, '1', 17),
        ('Ditto','2022-05-14', 4, '1', 22);
+
+INSERT INTO owners(full_name, age)
+VALUES ('Sam Smith', 34),
+('Jennifer Orwell', 19),
+('Bob', 45),
+('Melody Pond', 77),
+('Dean Winchester', 14),
+('Jodie Whittaker', 38);
+
+INSERT INTO species(name) VALUES ('Pokemon'), ('Digimon');
+
+UPDATE animals SET species_id = (SELECT id FROM species WHERE name = 'Digimon') WHERE name LIKE '%mon%';
+
+UPDATE animals SET species_id = (SELECT id FROM species WHERE name = 'Pokemon') WHERE name NOT LIKE '%mon%';
```

Comment:
- [ ] Consider updating animals that end with `mon`, so it will be `%mon` not `%mon%`.